README.md

```diff
@@ -66,6 +66,24 @@
 
 ## Installation
 
+### Swift Package
```

Comment:
I'd maybe add a `(Recommended)` or `(Recommended Way`) to the title.

Reply:
Done :)